diamm/serializers/website/source.py

```diff
@@ -68,11 +68,14 @@ def get_related_entity(self, obj):
 
 
 class SourceProvenanceSerializer(ContextDictSerializer):
+    entity = serpy.MethodField()
+    protectorate = serpy.MethodField()
+    region = serpy.MethodField()
     city = serpy.MethodField()
     country = serpy.MethodField()
-    region = serpy.MethodField()
-    protectorate = serpy.MethodField()
-    entity = serpy.MethodField()
+
+    class Meta:
+        ordering = ('entity', 'protectorate', 'region', 'city', 'country')
```

Comment:
So the issue is in the actual Dict-to-JSON conversion. Python dictionaries are unordered, so you're never guaranteed an ordered output. We use the `ujson` library which also does not order the dictionary, as far as I can remember.

There is an `OrderedDict` class available. If you really want ordered output it is possible to override the serialized output and replace it with an OrderedDict. See:

https://github.com/clarkduvall/serpy/pull/4
https://github.com/esnme/ultrajson/issues/55